titus-server-master/src/main/java/com/netflix/titus/master/scheduler/TaskPlacementFailure.java

```diff
@@ -65,7 +67,21 @@
          */
         OpportunisticResource,
 
-        Unrecognized,
+        Unrecognized;
+
+        /**
+         * Failures that are expected to go away, and should not be acted upon
+         *
+         * @see DefaultSchedulingService
+         */
+        public static final Set<FailureKind> TRANSIENT = Sets.immutableEnumSet(WaitingForInUseIpAllocation, LaunchGuard);
```

Comment:
Just pointing out that, while highly unlikely, resolution of this condition may be up to user if they are doing something strange so how long this scenario may exist is variable (e.g., we're expecting the user to terminate the the task holding the contended IP Allocation).

Reply:
Makes sense. I think it is still right to call it a "transient" failure, and that we don't need to act on it, correct?

Reply:
Yup. Just an FYI.